kingpin/actors/aws/ecs.py

```diff
@@ -1045,7 +1045,25 @@ def _update_service(self, service_name, existing_service, override=None):
             update_parameters.update(override)
 
         self.log.info('Updating service.')
-
+        self.log.info('Checking if existing service is active...')
+        service_is_active = (existing_service and
```

Comment:
Better to validate `existing_service` parameter is not None (and throw an exception) at the top of the function, and then assume it is ok for the rest of the function body.